tests/environment/environment.go

```diff
@@ -0,0 +1,13 @@
+package environment
+
+import (
+	"os"
+)
+
+func GetVarWithDefault(name string, defaultValue string) string {
+	value := os.Getenv(name)
```

Comment:
Shouldn't we handle or propagate the error `Getenv` return?

Reply:
There is no error at `Getenv` https://golang.org/pkg/os/#Getenv